fifteen_min/src/find_amenities.rs

```diff
@@ -48,9 +55,21 @@ fn create_multi_isochrone(
     Isochrone::new(ctx, app, stores, options)
 }
 
+/// Draw an isochrone from every intersection border
+fn create_border_isochrone(ctx: &mut EventCtx, app: &App, options: Options) -> BorderIsochrone {
+    let mut all_intersections = Vec::new();
+    for i in app.map.all_intersections() {
+        if i.is_border() {
+            all_intersections.push(i.id);
+        }
+    }
+    BorderIsochrone::new(ctx, app, all_intersections, options)
+}
+
 struct Results {
     draw: Drawable,
     isochrone: Isochrone,
+    border_isochrone: BorderIsochrone,
```

Comment:
Rust warns that this is never used. And indeed, we just use it for drawing the cloud of uncertainty. So we could remove it, unless you imagine a reason we'd want to use any of the info here while showing the results for amenity search